ckanext/validation/tests/test_form.py

```diff
@@ -405,9 +405,8 @@ def test_resource_form_create_invalid(self, app):
         )
 
         assert "validation" in response.body
-        assert "missing-cell" in response.body
-        assert 'Row at position \\&#34;2\\&#34; has a missing cell in field \\&#34;d\\&#34; at position \\&#34;4\\&#34;' in response.body
-        assert "This row has less values compared to the header row" in response.body
+        assert 'The form contains invalid entries' in response.body
+        assert ("There are validation issues with this file" in response.body) or ("Validation: errors" in response.body)
```

Comment:
Surely this messsage is to generic? We want to test that the message in clear and specific, no?